composer.json

```diff
@@ -13,7 +13,7 @@
         "irc": "irc://irc.freenode.net/phergie"
     },
     "require": {
-        "php": "^5.5||^7.0",
+        "php": "^5.6||^7.0",
```

Comment:
Thanks for noting this difference.  I meant to drop a comment in the PR when I submitted it, but I was fitting it in during work, like a slacker!